.github/workflows/static-test.yml

```
@@ -0,0 +1,21 @@
+name: static-test
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Run static test
```

You might need to add the variable `CI_BASE_BRANCH` for master here.
```suggestion
    - name: Run static test
      env:
        CI_BASE_BRANCH: master
```

---

I think I know the issue. I have to update the riot-action :)

---

Ok [that](https://github.com/aabadie/riot-action/commit/9cb06d1de106f9b8ddf75aaf151829fa23ab65cd) doesn't work either.